sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOSiteFileVersionBatchDeleteJob.md

```diff
@@ -16,7 +16,7 @@ manager: seanmc
 ## SYNOPSIS
 
 > [!NOTE]
-> This feature is part of the Version history controls Preview. If your tenant is not part of the Preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
+> This feature is part of the version history controls preview. If your tenant is not part of the preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
 
 Cancels further processing of a file version batch trim job for a site collection.
```

Comment:
Stops further processing of Site level trim job that is in-progress. 

Once the cmdlet executes successfully, all new asynchronous version deletion will be stopped. Stopping a trim job will not impact versions that have already been permanently deleted when the job was in progress.

Reply:
Just added "Stops further processing of Site level trim job that is in-progress." in the synopsis, added all in the description, please double check if it is ok.